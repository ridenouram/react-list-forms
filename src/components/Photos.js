import React from 'react';
import PropTypes from 'prop-types';
import Photo from './Photo';
import styles from './style.css';

export default function Photos({ photos }) {
  const photosList = photos.map(photo => {
    return (
      <li className={styles.li}key={photo}>
        <Photo photo={photo} />
      </li>
    );
  });

  return (
    <ul className={styles.Photos}>
      {photosList}
    </ul>
  );
}

Photos.propTypes = {
  photos: PropTypes.arrayOf(PropTypes.string.isRequired).isRequired
};
