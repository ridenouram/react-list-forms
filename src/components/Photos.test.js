import React from 'react';
import { shallow } from 'enzyme';
import Photos from './Photos';

describe('photos component', () => {
  it('matches snapshot', () => {
    const wrapper = shallow(<Photos photos={[]} />);
    expect(wrapper).toMatchSnapshot();
  });
});
