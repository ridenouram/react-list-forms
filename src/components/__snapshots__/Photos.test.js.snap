// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`photos component matches snapshot 1`] = `
<ul
  className="Photos"
/>
`;
